tutorials/quickstart-ruby.md

```diff
@@ -0,0 +1,443 @@
+file with the following to instruct the database to load the TimescaleDB 
+extension to PostgreSQL:
+
+```ruby
+class AddTimescale < ActiveRecord::Migration[5.2]
+  def change
+    enable_extension("timescaledb") unless extensions.include? "timescaledb"
+  end
+end
+```
+
+#### Step 2: Create the database
+
+Now we can run the following `rake` command to add the TimescaleDB extension
+to our database:
+
+```bash
+rake db:migrate
+```
+
+>:WARNING: In order for the command to work, you will need to make sure there is a database named `postgres` in your TimescaleDB deployment. This database is not present by default with Timescale Cloud.
+
+In `psql` you can test that the extension has been added by running the `\dx`
+command and seeing something like the following:
+
+```
+                                      List of installed extensions
+    Name     | Version |   Schema   |                            Description                            
+-------------+---------+------------+-------------------------------------------------------------------
+ plpgsql     | 1.0     | pg_catalog | PL/pgSQL procedural language
+ timescaledb | 1.6.1   | public     | Enables scalable inserts and complex queries for time-series data
+(2 rows)
+```
+
+#### Step 3: Create a table 
+
+Suppose we wanted to create a table to store the user agent (browser) and time 
+whenever a visitor loads our page. You could easily extend this simple example 
+to store a host of additional web analytics of interest to you. We can generate 
+a Rails scaffold to represent this information in a table:
+
+```bash
+rails generate scaffold PageLoads user_agent:string time:timestamp
+```
+
+Rails generates all the helper files and a database migration. We can then run 
+a `rake` command to create the table in our database.
+
+```bash
+rake db:migrate
+```
+
+If we connect to our Timescale instance using `psql`, we can view our database 
+and the `page_loads` table that was created through our migration process using 
+the `\dt` command:
+
+```bash
+                 List of relations
+ Schema |         Name         | Type  |   Owner   
+--------+----------------------+-------+-----------
+ public | ar_internal_metadata | table | tsdbadmin
+ public | page_loads           | table | tsdbadmin
+ public | schema_migrations    | table | tsdbadmin
+(3 rows)
+```
+
+And we can view the structure of the `page_loads` table using 
+the `\d page_loads` command:
+
+```bash
+                                        Table "public.page_loads"
+   Column   |            Type             | Collation | Nullable |                Default                 
+------------+-----------------------------+-----------+----------+----------------------------------------
+ id         | bigint                      |           | not null | nextval('page_loads_id_seq'::regclass)
+ user_agent | character varying           |           |          | 
+ time       | timestamp without time zone |           |          | 
```

Comment:
timestamp without time zone is bad practice and should use timestamp with time zone instead